ldap/servers/slapd/pw.h

```diff
@@ -41,7 +41,10 @@ int check_pw_duration_value(const char *attr_name, char *value, long minval, lon
 int check_pw_resetfailurecount_value(const char *attr_name, char *value, long minval, long maxval, char *errorbuf, size_t ebuflen);
 int check_pw_storagescheme_value(const char *attr_name, char *value, long minval, long maxval, char *errorbuf, size_t ebuflen);
 
-int pw_is_pwp_admin(Slapi_PBlock *pb, struct passwordpolicyarray *pwp);
+int pw_is_pwp_admin(Slapi_PBlock *pb, struct passwordpolicyarray *pwp, int ignore_rootdn);
+#define PWP_INCLUDE_ROOTDN 0
+#define PWP_IGNORE_ROOTDN 1
+
```

Comment:
My understanding is that this flag is used to check if the requestor is root || pwp_admin (INCLUDE_ROOTDN) or pwp_admin only.
I have no strong opinion but I would prefer something like PWP_ADMIN_ONLY and PWP_ADMIN_OR_ROOT

Reply:
Ok, I'll rewrite it